man/flac.md

```diff
@@ -125,64 +125,69 @@ Some common **encoding** tasks using flac:
 
 `flac --delete-input-file -w abc.wav`
-:	Like above, except abc.wav is deleted if there were no errors or warnings.
+:	Like above, except abc.wav is deleted if there were no errors and no warnings.
 
-`flac --best abc.wav`
-:	Encode abc.wav to abc.flac using the highest compression setting.
+`flac --best abc.wav` or `flac -8 abc.wav`
+:	Encode abc.wav to abc.flac using the highest compression preset.  
 
-`flac --verify abc.wav`
+`flac --verify abc.wav` or `flac -V abc.wav`
 :	Encode abc.wav to abc.flac and internally decode abc.flac to make sure it matches abc.wav.
 
 `flac -o my.flac abc.wav`
 :	Encode abc.wav to my.flac.
 
 `flac -T "TITLE=Bohemian Rhapsody" -T "ARTIST=Queen" abc.wav`
-:	Encode abc.wav and add some tags at the same time to abc.flac.
+:	Encode abc.wav and add some tags at the same time, to abc.flac.
 
-`flac *.wav`
-:	Encode all .wav files in the current directory.
+`flac abc.aiff foo.rf64 bar.w64`
+:	Encode abc.aiff to abc.flac, foo.rf64 to foo.flac and bar.w64 to bar.flac
 
-`flac abc.aiff`
-:	Encode abc.aiff to abc.flac.
+`flac *.wav *.aif`
+:	Encode all .wav files and all .aif files in the current directory.
 
-`flac abc.rf64`
-:	Encode abc.rf64 to abc.flac.
-
-`flac abc.w64`
-:	Encode abc.w64 to abc.flac.
-
-`flac abc.flac --force`
+`flac abc.flac --force` or `flac abc.flac -f`
 :	This one's a little tricky: notice that flac is in encode mode by
 	default (you have to specify -d to decode) so this command actually
 	recompresses abc.flac back to abc.flac. \--force is needed to make
 	sure you really want to overwrite abc.flac with a new version. Why
 	would you want to do this? It allows you to recompress an existing
 	FLAC file with (usually) higher compression options or a newer
 	version of FLAC and preserve all the metadata like tags too.
+ 	Note: If the FLAC file does not end with .flac - say, it is abc.fla
+  	- then a new abc.flac will be created and the old kept, just like 
+   	for an uncompressed input file.
+
+`flac -8Vfo Track04.flac -T "ARTIST=Queen" -- -4.wav`
+:	Options in their short version - those invoked with single dash - 
+	can be concatenated until one that takes an argument: here -o and 
+ 	-T do, and so one of them requires its own dash. 
+	This command line takes the track -4.wav as input; its filename 
+ 	starts with a dash, so it needs the \-- to keep the encoder from
+  	taking the "4" as an option. All options must go before the --: 
+        It will overwrite any Track04.flac already present, encode with
+	highest compression preset, decode to verify that it matches, and 
+ 	a tag will be added. 
+
 
 Some common **decoding** tasks using flac:
 
-`flac -d abc.flac`
+`flac --decode abc.flac` or `flac -d abc.flac`
 :	Decode abc.flac to abc.wav. abc.flac is not deleted. NOTE: Without 
-	-d it means re-encode abc.flac to abc.flac (see above).
-
-`flac -d --force-aiff-format abc.flac`  
+	-d it would mean to re-encode abc.flac to abc.flac (see above) and 
```

Comment:
This is a little inconsistent: you've changed `-d`to `--decode` in the command, but not the explanation